pretixprint/app/src/main/res/xml/preferences.xml

```diff
@@ -33,6 +33,13 @@
             />
     </PreferenceCategory>
 
+    <PreferenceCategory android:title="@string/settings_label_security">
+        <eu.pretix.pretixprint.ui.ProtectedEditTextPreference
+            android:inputType="number"
+            android:key="pref_pin"
+            android:title="@string/pin_protection_pin" />
```

Comment:
Bytes are cheap - let's add a little description what does and why you'll be annoyed when you forget the PIN ;-)